packages/vant-use/src/useRaf/index.ts

```diff
@@ -0,0 +1,28 @@
+import { inBrowser } from '..';
+
+export function useRaf(fn: FrameRequestCallback, interval = 0): () => void {
+  if (inBrowser) {
+    let start: number;
+    const disposesId: number[] = [];
+    let isStopped = false;
+
+    const frameWrapper = (timestamp: number) => {
+      if (isStopped) return;
+      if (start === undefined) {
+        start = timestamp;
+      } else if (timestamp - start > interval) {
+        fn(timestamp);
+        start = timestamp;
+      }
+      disposesId.push(requestAnimationFrame(frameWrapper));
```

Comment:
The disposesId will only be cleared when cancel is called. Considering the execution frequency of requestAnimationFrame, the length of disposesId should increase rapidly.

Reply:
What I originally thought was to clear all requestAnimation, but now it seems that there will be performance problems, and the function in requestAnimationFrame should return directly without any burden

Reply:
I have fixed it. Before the next loop, it will cancel the pre